plugins/generator-1.14.4/forge-1.14.4/procedures/block_fluid_to_bucket.java.ftl

```diff
@@ -0,0 +1,3 @@
+<#include "mcitems.ftl">
+(${mappedBlockToBlockStateCode(input$block)}.getBlock() instanceof FlowingFluidBlock ?
+((FlowingFluidBlock) ${mappedBlockToBlockStateCode(input$block)}.getBlock()).getFluid().getFilledBucket() : Items.AIR)
```

Comment:
Return ItemStack, not Item. Your code might work, but not in all cases since the convention is to use ItemStacks. Also, check this in the 1.15.2 counterpart.